pkg/processor/build/types.go

```diff
@@ -0,0 +1,31 @@
+/*
+Copyright 2017 The Nuclio Authors.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package build
+
+type Options struct {
+	FunctionName    string
+	FunctionPath    string
+	NuclioSourceDir string
+	NuclioSourceURL string
+	PushRegistry    string
+	Runtime         string
+	Verbose         bool
+	OutputName      string
+	OutputType      string
+	OutputVersion   string
+	NoBaseImagePull bool
```

Comment:
Maybe `DoBaseImagePull` (reversed logic) is a better option.

Reply:
True, but I want default value to be the common case (so in most cases this should be false)